google/cloud/forseti/common/gcp_api/repository_mixins.py

```diff
@@ -358,6 +358,27 @@ def create(self, verb='create', **kwargs):
         return self.execute_query(verb=verb, verb_arguments=arguments)
 
 
+class PatchQueryMixin(object):
```

Comment:
There is an existing PatchResourceMixin class you can use, please switch to that.

Reply:
thanks!